Resources/Prototypes/Recipes/Construction/Graphs/machines/cyborg.yml

```diff
@@ -179,6 +179,34 @@
       - tool: Screwing
         doAfter: 0.5
 
+
+    - to: security # Delta-V Security Borg Start
+      steps:
+      - assemblyId: security
+        guideString: borg-construction-guide-string
+
+      - material: Cable
+        amount: 1
+        doAfter: 1
+        store: part-container
+
+      - component: Flash
+        name: flash
+        store: part-container
+        icon:
+          sprite: Objects/Weapons/Melee/flash.rsi
+          state: flash
+
+      - component: Flash
```

Comment:
Change this to something security specific, so epi traitors cant just build these and emag them for free sec access. 
Baton would be good.

Reply:
resolved